waiter/test/waiter/scaling_test.clj

```diff
@@ -667,45 +667,95 @@
+                                 ["app1" "app2" "app3" "app4" "app5"]
+                                 ;; service-id->service-description
                                  {"app1" (merge config {})
                                   "app2" (merge config {})
                                   "app3" (merge config {"min-instances" 5})
-                                  "app4" (merge config {"max-instances" 10})} ; service description
+                                  "app4" (merge config {"max-instances" 10})
+                                  "app5" (merge config {"max-instances" 20})
+                                  "app6" (merge config {})}
+                                 ;; service-id->outstanding-requests
                                  {"app1" 10
                                   "app2" 5
                                   "app3" 0
-                                  "app4" 15} ; outstanding requests
+                                  "app4" 15
+                                  "app5" 12
+                                  "app6" 10}
+                                 ;; service-id->scale-state
                                  {"app1" {:target-instances 5}
                                   "app2" {:target-instances 5}
                                   "app3" {:target-instances 0}
-                                  "app4" {:target-instances 10}} ; scale state
-                                 apply-scaling 5 test-scale-service
-                                 {"app1" {:healthy-instances 5 :task-count 5 :expired-instances 0}
-                                  "app2" {:healthy-instances 5 :task-count 5 :expired-instances 0}
-                                  "app3" {:healthy-instances 0 :task-count 0 :expired-instances 0}
-                                  "app4" {:healthy-instances 15 :task-count 15 :expired-instances 0}}
+                                  "app4" {:target-instances 10}
+                                  "app5" {:target-instances 12}
+                                  "app6" {:target-instances 10}}
+                                 apply-scaling
+                                 5
+                                 test-scale-service
+                                 ;; service-id->router-state
+                                 {"app1" {:healthy-instances 5 :task-count 5 :expired-healthy-instances 0 :expired-unhealthy-instances 0}
+                                  "app2" {:healthy-instances 5 :task-count 5 :expired-healthy-instances 0 :expired-unhealthy-instances 0}
+                                  "app3" {:healthy-instances 0 :task-count 0 :expired-healthy-instances 0 :expired-unhealthy-instances 0}
+                                  "app4" {:healthy-instances 15 :task-count 15 :expired-healthy-instances 0 :expired-unhealthy-instances 0}
+                                  "app5" {:healthy-instances 10 :task-count 10 :expired-healthy-instances 7 :expired-unhealthy-instances 4}
```

Comment:
If there are 10 healthy and 4 expired unhealthy, why is the task count 10?

Reply:
Changed to 14.